libbeat/docs/outputconfig.asciidoc

```diff
@@ -930,47 +938,67 @@ The list of root certificates for server verifications. If `certificate_authorit
 
 ===== certificate: "/etc/pki/client/cert.pem"
 
-The path to the certificate for TLS client authentication. If the certificate
+The path to the certificate for SSL client authentication. If the certificate
 is not specified, client authentication is not available. The connection
-might fail if the server requests client authentication. If the TLS server does not
+might fail if the server requests client authentication. If the SSL server does not
 require client authentication, the certificate will be loaded, but not requested or used
 by the server.
 
 When this option is configured, the <<certificate_key>> option is also required.
 
 [[certificate_key]]
-===== certificate_key: "/etc/pki/client/cert.key"
+===== key: "/etc/pki/client/cert.key"
 
 The client certificate key used for client authentication. This option is required if <<certificate>> is specified.
 
-===== min_version
+===== key_passphrase
 
-The minimum SSL/TLS version allowed for the encrypted connections. The value must be one of the following:
-`SSL-3.0` for SSL 3, `1.0` for TLS 1.0, `1.1` for TLS 1.1 and `1.2` for TLS 1.2.
+The passphrase used to decrypt an encrypted key stored in the configured `key` file.
 
-The default value is `1.0`.
+===== versions
 
-===== max_version
+List of allowed SSL/TLS versions. If SSL/TLS server decides for protocol verions
+not allowed, the connection will be dropped during or after the handshake. The
+setting is a list setting the allowed protocol versions:
```

Comment:
Use either "is a list setting that specifies the allowed protocol versions" or "is a list of allowed protocol versions".